.gitignore

```diff
@@ -23,6 +23,9 @@ parcel-bundle-reports
 
 # node_modules
 node_modules
+package-lock.json
```

Comment:
I feel like there is something wrong here

Reply:
I am unsure why you wrote `package-lock.json `and` !/package-lock.json`

Reply:
to prevent other `package-lock.json` from being committed as there should only be one `package-lock.json` at the root.